packages/app/src/views/Home.vue

```diff
@@ -5,9 +5,20 @@
         <course-search-bar v-if="!$apollo.loading" :courses="formattedCourses" />
       </v-col>
     </v-row>
-    <v-row>
+    <v-row style="background: #E5E5E5;">
+      <v-col>
+        <timetable :timetable="timetable" />
+      </v-col>
+    </v-row>
+    <v-row style="background: #E5E5E5;">
       <v-col>
-        <timetable :timetable="timetable"/>
+        <TimetableCourseCard
+          v-for="(info, course) in courseCards"
```

Comment:
After renaming courseCards to courses, we can make this v-for make more sense by writing v-for="(course, code) in courses"